chrome/content/brief-overlay.js

```diff
@@ -135,14 +135,11 @@ var Brief = {
             }.bind(this))
         }
 
-        if (this.toolbarbutton)
```

Comment:
I don't know the exact reason for these checks you remove. Are we sure it's OK to remove them?